mistletoe/html_renderer.py

```diff
@@ -18,13 +18,19 @@ class HTMLRenderer(BaseRenderer):
 
     See mistletoe.base_renderer module for more info.
     """
-    def __init__(self, *extras):
+    def __init__(self, *extras, html_escape_double_quotes=True, html_escape_single_quotes=False):
```

Comment:
"html-escape" or just "escape"? Imho, escape is sufficient.

Reply:
I was asking myself the same question. :) And I decided to go with the longer variant, so I can be sure this won't collide for example with `latex_escape_double_quotes` or similar in the future, no matter how improbable this is now. ;)